python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -828,6 +828,57 @@ export default function EditorContainer({
     return () => clearInterval(interval);
   }, [getServerStatusCallback, serverStatus]);
 
+  const runningPromptId: string | undefined = aiconfigState._ui.runningPromptId;
+  const promptListContainer = (
```

Comment:
Also I know this is a very big file and we can fix that down the line:
- pulling our callback definitions into their own hooks
- using context instead of passing a bunch of props through here (but I want us to align on whether we need to support custom react components before doing that)